tradeit-android-sdk/src/main/java/it/trade/android/sdk/model/TradeItLinkedBrokerParcelable.java

```diff
@@ -255,6 +255,16 @@ public boolean equals(Object o) {
         return linkedLogin.userId.equals(that.linkedLogin.userId);
     }
 
+    public boolean equalsAccounts(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+
+        TradeItLinkedBrokerParcelable that = (TradeItLinkedBrokerParcelable) o;
+
+        return linkedLogin.userId.equals(that.linkedLogin.userId) &&
```

Comment:
- label - No, I don't think this is really used anywhere.
- broker - No, this shouldn't change ever.
- accountLastUpdated - No, not a big deal.
- error - No, this gets reset every time the app loads anyways.
- userToken - Yes. For Yahoo this should never change because they aren't returning userId to the app, but any other future partners that use the syncing will need to update LB to the new userToken.